.golangci.yml

```diff
@@ -76,6 +76,8 @@ linters-settings:
       - stdlib
       # also allow generics
       - generic
+      # we want to return spi.Interfaces in routing.go
```

Comment:
I think the comment needs to be updated.

Reply:
I was thinking about something more generic, as we might rename the files on the way, but that is fine for now :)